cli/jobs/train/r/basic-train-model/data/iris.csv

```diff
@@ -0,0 +1,151 @@
+"Sepal.Length","Sepal.Width","Petal.Length","Petal.Width","Species"
```

Comment:
don't put data in the repo, use cloud data - see other examples for public iris url

Reply:
couldn't get the file mounting to work -- also, showing how to use local data matches more the users' use cases (rather than public internet data).

Reply:
Azure users and the enterprise ML professionals we target are expected to have their data in the cloud - the only difference is the authentication, which we do need to have a good story and samples around. I don't think leading users in the direction of uploading data from local for runs in the cloud is a good pattern